package searchapi;

message Request {
    required string query = 1;
}

message Response {
    required string result = 1;
    optional string error = 2;
    optional uint64 result_code = 3;
}

service RpcService {
    rpc RpcFunc (Request) returns (Response);
}